src/components/BlogPost/BlogPost.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardActionArea, CardContent, CardMedia, Typography, Chip } from '@material-ui/core';
+
+export interface IBlogPost {
+  title: string;
+  text: string;
+  tagName: string;
+  date: string;
+  image: string;
+}
+
+const useStyles = makeStyles(({ spacing, typography }) => ({
+  root: {
+    maxWidth: 400,
+  },
+  media: {
+    height: 224,
+    position: 'relative',
+  },
+  content: {
+    padding: spacing(3),
+  },
+  title: {
+    margin: 0,
+  },
+  text: {
+    paddingTop: spacing(1.5),
+  },
+  tag: {
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    borderRadius: spacing(0, 1, 0, 0),
```

Comment:
Don't use spacing for anything else than padding, margin, and non-static positioning